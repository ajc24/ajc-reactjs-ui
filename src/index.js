export { default as Footer } from './ui-elements/Footer';
export { default as Header } from './ui-elements/Header';
export { default as HeaderLogo } from './ui-elements/HeaderLogo';
export { default as HeaderTitleAndSubtitle } from './ui-elements/HeaderTitleAndSubtitle';
export { default as MainContentPageTitle } from './ui-elements/MainContentPageTitle';
export { default as MainContentPanel } from './ui-elements/MainContentPanel';
export { default as MainContentText } from './ui-elements/MainContentText';
export { default as Menu } from './ui-elements/Menu';
export { default as Screenshot } from './ui-elements/Screenshot';
export { default as SmallMenuIcon } from './ui-elements/SmallMenuIcon';
export { default as SmallRightArrowIcon } from './ui-elements/SmallRightArrowIcon';
