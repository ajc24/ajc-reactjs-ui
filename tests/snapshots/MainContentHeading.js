import React from 'react';
import { TestDev } from 'ajc-jest-enzyme';
import { MainContentHeading } from '../../src';

describe('MainContentHeading', () => {
  describe('Default props and rendering', () => {
    let jsonSnapshot;

    beforeAll(() => {
      jsonSnapshot = TestDev.createSnapshot(
        <React.Fragment>
          <MainContentHeading pageHeading="Test Page Heading" />
        </React.Fragment>
      );
    });
    
    it('verifies the snapshot for the component', () => {
      expect(jsonSnapshot).toMatchSnapshot();
    });
  });
});
