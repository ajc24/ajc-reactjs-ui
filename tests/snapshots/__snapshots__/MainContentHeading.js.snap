// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContentHeading Default props and rendering verifies the snapshot for the component 1`] = `
<div
  className="ajc-main-content-heading-container ajc-text-black ajc-default"
>
  <h1
    className="ajc-main-content-heading-text"
  >
    Test Page Heading
  </h1>
  <div
    className="ajc-main-content-heading-divider-outer"
    role="presentation"
  >
    <div
      className="ajc-main-content-heading-divider-inner"
    />
  </div>
</div>
`;
